rust/abacus-base/src/contract_sync/interchain_gas.rs

```diff
@@ -83,7 +73,6 @@ where
                 stored_messages.inc_by(new_payments_processed);
 
                 db.store_latest_indexed_gas_payment_block(to)?;
```

Comment:
Ah, yeah, okay, this is an easy enough fix. My gut reaction though here is to use start_block and not `from` since it is the "full chunk from" and that could technically lead to us backtracking. I think we might also want to change that in the outbox to not use full chunk from but rather the `start_block`. Thoughts?

Reply:
sorry I don't think I follow, what are you suggesting we use `start_block` for?